### PR TITLE
feat: Support fallback options in trtorchc

### PR DESCRIPTION
**Pull request by peri044**
# Description

Support fallback options in trtorchc

## Type of change

- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: cpp/trtorchc/README.md [hunk 1]
```diff
@@ -37,13 +37,17 @@ trtorchc [input_file_path] [output_file_path]
       --allow-gpu-fallback              (Only used when targeting DLA
                                         (device-type)) Lets engine run layers on
                                         GPU if they are not supported on DLA
+      --allow-torch-fallback            Enable layers to run in torch
```
Comment (narendasan) [inline cpp/trtorchc/README.md:40]: We need to add these docs to the trtorchc docs page too

File: cpp/trtorchc/main.cpp [hunk 2]
```diff
@@ -191,6 +197,11 @@ int main(int argc, char** argv) {
       "file_path",
       "Path to calibration cache file to use for post training quantization",
       {"calibration-cache-file"});
+  args::ValueFlag<std::string> forced_fallback_ops(
```
Comment (narendasan) [inline cpp/trtorchc/main.cpp:200]: I think the args package we use allows for repeated args might help avoid string parsing

Comment (narendasan) [reply cpp/trtorchc/main.cpp:200]: Look at the input PR for how to use this

File: cpp/trtorchc/main.cpp [hunk 3]
```diff
@@ -375,11 +402,6 @@ int main(int argc, char** argv) {
     return 1;
   }
 
-  if (!trtorch::CheckMethodOperatorSupport(mod, "forward")) {
```
Comment (narendasan) [inline cpp/trtorchc/main.cpp:378]: Why was this removed?

Comment (peri044) [reply cpp/trtorchc/main.cpp:378]: Fixed it now. CheckMethodOperatorSupport errors out listing all the unsupported ops. When torch_fallback is enabled, this will stop execution without even going to CompileGraph which shouldn't happen.

**Comment by narendasan** [comment]
@peri044 can you rebase on master?
